src/Umbraco.Cms.ManagementApi/Controllers/Language/GetAllLanguageController.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Umbraco.Cms.Core.Mapping;
+using Umbraco.Cms.Core.Models;
+using Umbraco.Cms.Core.Services;
+using Umbraco.Cms.ManagementApi.ViewModels.Languages;
+using Umbraco.Cms.ManagementApi.ViewModels.Pagination;
+using Umbraco.New.Cms.Core.Models;
+
+namespace Umbraco.Cms.ManagementApi.Controllers.Language;
+
+[ApiVersion("1.0")]
```

Comment:
Since all endpoints a v1 you can move this attribute to the base controller, then you don't have to have it in each action controller